lib/browser/api/dialog.js

```diff
@@ -64,203 +44,140 @@ const normalizeAccessKey = (text) => {
   return text
 }
 
-const checkAppInitialized = function () {
+const checkAppInitialized = () => {
   if (!app.isReady()) {
     throw new Error('dialog module can only be used after app is ready')
   }
 }
 
 module.exports = {
-  showOpenDialog: function (...args) {
+  showOpenDialog: opts => {
     checkAppInitialized()
 
-    let [window, options, callback] = parseArgs(...args)
+    // destructure params with default options value
+    let {window, options = {title: 'Open', properties: ['openFile']}, callback} = opts
```

Comment:
Maybe you don't need default here? If you set them again a few lines after?

Reply:
Also these options won't deep default.

I.e.

```js
const foo = (opts = { a: 1, b: 2}) => return opts;

foo(); // { a: 1, b: 2 }
foo({ a: 3 }); // { a: 3 } (no b)
```